Formula/fn.rb

```diff
@@ -0,0 +1,23 @@
+class Fn < Formula
+  desc "IronFunctions command-line tools"
+  homepage "https://iron.io"
+
+  stable do
```

Comment:
No need to put this in a `stable do` block if there is no corresponding `devel do` or `head do` block.